docs/source/monitors.rst

```diff
@@ -52,38 +168,149 @@ a function of time.
 The OSM Shift monitors are broken up into FUV and NUV components and are also tracked both for the along-dispersion
 (SHIFT1) and cross-dispersion (SHIFT2) shifts for a total of four individual monitors.
 
+FUV OSM Shift Monitors
+++++++++++++++++++++++
 Tracking
-++++++++
+........
 For FUV, the OSM Shift monitors track the difference between the reported shift for the two FUV segments, FUVA and FUVB,
 in the form of FUVA - FUVB.
-NUV tracking TBD.
+Outliers for the SHIFT1 and SHIFT2 Monitors are those exposures with a segment difference (FUVA - FUVB) of greater than
+10 pixels and greater than 5 pixels respectively.
 
-.. need to add more about why we want to look at A - B
 
 Output
-++++++
-FUV output for both SHIFT1 and SHIFT2 monitors consist of the shift measurement (gathered from ``lampflash`` files)
-plotted as a function of time.
-Each grating/cenwave combination is plotted as a different color, and each FPPOS is plotted with a different symbol.
-Additionally there are button options to switch between viewing the shift vs time for all FPPOS and individual FPPOS.
+......
+FUV output for both SHIFT1 and SHIFT2 monitors consist of two subplots:
+
+- The shift measurement plotted as a function of time
+- The segment difference plotted as a function of time
+
+.. note::
+
+    The FPPOS offset is *not* removed from the SHIFT1/SHIFT2 value.
+    This is intentional so that trends per FPPOS can be directly compared against each other.
+
+Each grating/cenwave combination is plotted as a different color, and each FPPOS is plotted with a different symbol
+(these individual elements can be selected/deselected via the legend).
+Exposures that occurred at LP3 after the move to LP4 are slightly enlarged.
+Outliers are indicated with red.
+
+There are button options to switch between viewing the shift vs time for all FPPOS and individual FPPOS.
 Vertical lines are included to denote the beginning of each new Lifetime Position.
 
-.. NUV will be added once the output is finalized
+Hover text for each data point includes the following::
+
+    # For the Shift vs Time subplot
+    (x, y) or (Datetime, Shift)
+    ROOTNAME
+    LIFE_ADJ
+    FPPOS
+    PROPOSID
+    SEGMENT
+    CENWAVE
+
+    # For the FUVA - FUVB vs Time subplot
+    (x, y) or (Datetime, A - B)
+    ROOTNAME
+    LIFE_ADJ
+    FPPOS
+    PROPOSID
+    SEGMENT
```

Comment:
Should Remove SEGMENT from this list (and the monitor itself)

Reply:
Filed issue #141 to resolve this. Will update the docs as a part of the fix